packages/design-system/src/organisms/layout/layout.stories.tsx

```diff
@@ -0,0 +1,185 @@
+import { fromTemplate, titleBuilder } from '../../../.storybook/utils'
+import MintingLogo from '../../assets/svg/minting-logo.svg'
+import SimpleLogo from '../../assets/svg/logoSimple.svg'
+import { TypographyV2 } from '../../atoms/typographyV2'
+import { JoinBanner, TokenAppearance } from '../../molecules'
+import {
+  discordInviteUrl,
+  faqUrl,
+  instaUrl,
+  resourceCenterUrl,
+  stakingTermsUrl,
+  twitterUrl,
+} from '../../constants'
+import { Web3ProviderWagmi } from '../../providers'
+import { CHAIN_ID_MAIN_NET } from '../../web3'
+import { Layout, LayoutProps } from './index'
+
+const storyConfig = {
+  title: titleBuilder.organisms('Layout'),
+  component: Layout,
+}
+
+const footerOptions = [
+  {
+    title: 'Twitter',
+    link: twitterUrl,
+  },
+  {
+    title: 'Discord',
+    link: discordInviteUrl,
+  },
+  {
+    title: 'Instagram',
+    link: instaUrl,
+  },
+  {
+    title: 'FAQ',
+    link: faqUrl,
+  },
+  {
+    title: 'Resourse Center',
+    link: resourceCenterUrl,
+  },
+  {
+    title: 'Protocol Terms',
+    link: stakingTermsUrl,
+  },
+]
+
+const Template = (props: LayoutProps) => (
+  <Web3ProviderWagmi
+    variant="web3Modal"
+    supportedChainIds={[CHAIN_ID_MAIN_NET]}
+    wallectConnectProjectId={'123'}
```

Comment:
Shouldn't this value be loaded for env variables?

Reply:
ah, yeah, leftover

Reply:
✅